.github/workflows/webpack-jumpstart.yml

```diff
@@ -11,16 +11,21 @@ jobs:
       pull-requests: none
     timeout-minutes: 20
     env:
-      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+      BASE_SHA: ${{ github.sha }}
+      HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
     steps:
       - name: Build jumpstart tarball
         run: |
           id="$(docker container create -u node -w /home/node node /bin/sh -c \
-            'git init cockpit &&
+            "set -x &&
+             git config --global user.email '<>' &&
+             git config --global user.name 'nobody' &&
+             git init cockpit &&
              cd cockpit &&
-             git fetch --depth 1 https://github.com/cockpit-project/cockpit "'"${MERGE_SHA}"'":build &&
-             git checkout build &&
-             pkg/build webpack-jumpstart.tar -j$(nproc)'
+             git fetch --no-tags https://github.com/cockpit-project/cockpit ${BASE_SHA}:base ${HEAD_SHA}:head &&
+             git checkout --detach ${BASE_SHA} &&
+             git merge --no-edit head &&
```

Comment:
We rebase in both integration tests (`make-checkout`) and for final landing. It would IMHO be nicer to consistently use rebase. AFAIK you don't really use the merge commit anywhere? That would also save you the `git config` dance.

Reply:
Rebase would even be nice because you can do it as a one-liner.

I use merge because GitHub uses merge and so will packit. 

Also, I still need the dance for rebase because of the committer email (and no option I could find to disable the rewriting).

I have a (probably irrational) concern about merges and rebasing producing different results.

Reply:
"GitHub uses merge" -- in what way? not in any that concerns us at least. Assuming that merging really could produce a different tree than rebasing, then we *really* want to rebase here, to be consistent with what happens in reality (CI and landing)

Reply:
GitHub refuses to rebase and merge if rebasing would produce a different result from merging.

I actually consider the fact that I intend to implement this via rebasing on the other side a feature: that way we'll find out if something changed.

I can try to implement this as rebase, but I also don't see any good reason in favour, and most of my "gut feeling" points to merge.

Reply:
..
and of course, I mean GitHub uses merges for running workflows.

Reply:
I just validated that `git merge --no-edit` fails with exit code 1 on conflicts. It leaves the tree in a complete mess (which we don't ever want to build -- if it succeeds, because the conflict is in the C part, we lose). So to make this work (regardless of rebase vs. merge), the container command needs to be invoked with the shell's `-e` option.